### ISSUE TITLE
Add more informative output to the force comparison in system_checker

### ISSUE THREAD

**Issue by davidlmobley** [choderalab/openmoltools#227]
`system_checker` provides some nice functionality for cross-comparing energies of `OpenMM` systems which should be identical. However, the current output is not particularly useful as it just gives things like this:

```
Error: (proper) PeriodicTorsionForce strength (9, 6, 8, 25) (3, 0.000000) has values of 0.697333 and 2.092000, respectively.
```

This provides very little help in tracking down which actual torsion parameter might be different.  I need to add more informative output.

**Comment by jchodera** [comment]
Would be great to make it clearer what those numbers mean.

**Comment by davidlmobley** [comment]
Agreed. If you have any handy info on how to retrieve that info I'd certainly appreciate it, as I'm still figuring out a lot about OpenMM. I can obviously adjust that specific statement so it specifies that the values are, er, the atoms involved in the torsion, the periodicity and phase, and the constants, but I don't have units...

My PR implements something which prints out which atoms these are (atom names), which is more helpful.

**Comment by jchodera** [comment]
Can you point me to the line that generates this?

**Comment by davidlmobley** [comment]
check_proper_torsions, starting here
https://github.com/choderalab/openmoltools/blob/master/openmoltools/system_checker.py#L354

On Mon, Jul 4, 2016 at 3:10 PM, John Chodera <notifications@github.com>
wrote:

> Can you point me to the line that generates this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/choderalab/openmoltools/issues/227#issuecomment-230358973>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGzUYamq3gec3CwOD-aHD6YUCHdjvWO-ks5qSYTsgaJpZM4JEnrF>
> .
>



-- 
David Mobley
dmobley@gmail.com
949-385-2436
